docs/content/dagster-cloud/account/authentication/okta/scim-provisioning.mdx

```diff
@@ -0,0 +1,185 @@
+- **Permissions in Okta that allow you to configure applications.**
+- **The following in Dagster Cloud:**
+  - An Enterprise plan
+  - [Organization Admin permissions](/dagster-cloud/account/managing-users/managing-user-roles-permissions#user-permissions-reference) in your organization
+
+---
+
+## Step 1: Enable SCIM provisioning in Dagster Cloud
+
+1. Sign in to your Dagster Cloud account.
+2. Click the **user menu (your icon) > Cloud Settings**.
+3. Click the **Provisioning** tab.
+4. If SCIM provisioning isn't enabled, click the **Enable SCIM provisioning** button to enable it.
+5. Click **Create SCIM token** to create an API token. This token will be used to authenticate requests from Okta to Dagster Cloud.
+
+Keep the API token handy - you'll need it in the next step.
+
+---
+
+## Step 2: Enable SCIM provisioning in Okta
+
+1. Sign in to your Okta Admin Dashboard.
+
+2. Using the sidebar, click **Applications > Applications**.
+
+3. Click the Dagster Cloud app. **Note**: If you haven't set up SSO for Okta, [follow this guide](/dagster-cloud/account/authentication/setting-up-okta-saml-sso)) to do so before continuing.
+
+4. Click the **Sign On** tab and complete the following:
+
+   1. Click **Edit**.
+
+   2. In the **Advanced Sign-on Settings** section, enter the name of your organization in the **Organization** field.
+
+   3. In the **Credential Details** section, set the **Application username format** field to **Email**:
+
+      <Image
+      alt="Configured Sign On tab of Dagster Cloud Okta application"
+      src="/images/dagster-cloud/sso/okta/scim-sign-on-tab.png"
+      width={800}
+      height={606}
+      />
+
+   4. Click **Save**.
+
+5. Click the **Provisioning** tab and complete the following:
+
+   1. Click **Configure API Integration**.
+
+   2. Check the **Enable API integration** checkbox that displays.
+
+   3. In the **API Token** field, paste the Dagster Cloud API token you generated in [Step 1](#step-1-enable-scim-provisioning-in-dagster-cloud):
+
+      <Image
+      alt="Configured Provisioning tab of Dagster Cloud Okta application"
+      src="/images/dagster-cloud/sso/okta/provisioning-tab.png"
+      width={1024}
+      height={649}
+      />
+
+   4. Click **Test API Credentials** to verify that your organization and API token work correctly.
+
+   5. When finished, click **Save**.
+
+---
+
+## Step 3: Enable user syncing in Okta
+
+After you confirm that your API credentials work in the Dagster Cloud Okta application, you can enable user syncing:
+
+1. In the Dagster Cloud Okta app, click the **Provisioning** tab.
+
+2. In the **Settings** panel, click **To App**.
+
+3. Click **Edit**.
+
+4. Next to **Create Users**, check the **Enable** checkbox. **Note**: The default username used to create accounts must be set to **Email** or user provisioning may not work correctly:
```

Comment:
nit "correctly." not "correctly:"?

Reply:
Not a nit - I did this stylistically as copy that precedes a related image. Coming back to this though I think it's just awkward, so I'll change it.